app/views/communicart_mailer/new_attachment_email.html.haml

```diff
@@ -0,0 +1,9 @@
+%div
+  %p
+    Hello,
+    %br
+    An attachment has been added to #{@proposal.name} (#{@proposal.public_id}).
+
+%p
+  %strong
+    = link_to "View This Request", proposal_url(@proposal), {target: 'C2'}
```

Comment:
The only thing that I would include is the title of the document. Can we add a line break after the "An attachment has been added" sentence and link to the doc using the attachment filename?